recipes/cost733class/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "cost733class" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lochbika/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dd883ac706c30da80184409b69d2196c1fa1da0073af62dffab434c1373d91a5
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+
+    # Unix
+    - {{ compiler('c') }}              # [unix]
+    - {{ compiler('cxx') }}            # [unix]
+    - {{ compiler('fortran') }}        # [unix]
+    - make                             # [unix]
+    - pkg-config                       # [unix]
+    
+    # Win
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - m2-make                          # [win]
+    - m2w64-pkg-config                 # [win]
+    - m2w64-dlfcn                      # [win]
+
+  host:
+    - netcdf-fortran                   # [not win]
+    - netcdf-fortran >=4.6             # [win]
+    - eccodes
+  run:
+    - netcdf-fortran                   # [not win]
+    - netcdf-fortran >=4.6             # [win]
+    - eccodes
+
+test:
+  commands:
+    - test -f $PREFIX/bin/cost733class                     # [unix]
+    - if not exist %LIBRARY_BIN%\\cost733class.exe exit 1  # [win]
+
+about:
+  home: https://projects.met.no/cost733/
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
+  summary: 'Program package for weather and circulation type classification.'
+  description: |
+    cost733class is a program package for weather and circulation type classification.
+    Copyright (C) 2013 Andreas Philipp
+  doc_url: https://git.rz.uni-augsburg.de/philipan/cost733class-1.4/-/raw/main/doc/cost733class_userguide.pdf
+  dev_url: https://github.com/lochbika/{{ name }}
```

Let's not abuse the jinja variable. It is nice to make this URL clickable with the proper name in it.

```suggestion
  dev_url: https://github.com/lochbika/cost733class
```